src/functions/indGraphSparse.jl

```diff
@@ -0,0 +1,56 @@
+struct IndGraphSparse{T <: RealOrComplex} <: IndGraph
+  m::Int
+  n::Int
+  A::AbstractArray{T,2}
+  F::Base.SparseArrays.CHOLMOD.Factor{T} #LDL factorization
+end
+
+function IndGraphSparse(A::AbstractArray{T,2}) where {T <: RealOrComplex}
+  m, n = size(A)
+  K = [speye(n) A'; A -speye(m)]
+
+  F = LinAlg.ldltfact(K)
+  #normrows = vec(sqrt.(sum(abs2.(A), 2)))
+
+  return IndGraphSparse(m, n, A, F)
+end
+
+# is_convex(f::IndGraph) = true
+# is_set(f::IndGraph) = true
+# is_cone(f::IndGraph) = false
+
+function prox!{T <: RealOrComplex}(
+    x::AbstractArray{T, 1},
+    y::AbstractArray{T, 1},
+    f::IndGraphSparse,
+    c::AbstractArray{T, 1},
+    d::AbstractArray{T, 1})
+  res = [c + f.A' * d; zeros(f.m)]
+  xy = f.F \ res
+  x[:] = xy[1:f.n]
+  y[:] = xy[end - f.m + 1:end]
```

Comment:
This might be a problem. Let us assume someone iterate with `indGraph` and `prox!` - someone like me ;)

So before I start I allocate `x` and `y` holders outside prox operator. If I understood your suggestions right - `view` variant lead to the situation when data physically stored inside `res` of `IndGraph` object. This is bad.

Regarding the `copy` variant - is there any difference between `x[:] .= xy[1:n]` and `copy!(x, 1, xy, 1, n)`? I assume yes, but can you point out the difference?